builtin/google/cloudrun/platform.go

```diff
@@ -383,7 +387,7 @@ func (p *Platform) Deploy(
 
 		service, err = deployment.createService(ctx, service)
 		if err != nil {
-			return nil, status.Errorf(codes.Aborted, "Unable to create Cloud Run service: %s", err.Error())
+			return status.Errorf(codes.Aborted, "Unable to create Cloud Run service: %s", err.Error())
```

Comment:
Nit: In this method sometimes we error out with `fmt.Errorf` and others like this we use `status.Errorf`. I realize you may not have been the author of this but I'm curious if you think we should use this chance to standardize on one?

Reply:
Makes sense to me! Done.